tests/system/api/test_artifacts.py

```diff
@@ -58,3 +61,73 @@ def test_fail_overflowing_artifact(self):
         assert (
             "Failed committing changes to DB" in run["status"]["error"]
         ), "run should fail with a reason"
+
+    def test_model_artifact_tags(self):
+        project_name = "my-test-project"
+        model_name = "test_model"
+        tag_1 = "tag_1"
+        tag_2 = "tag_2"
+        models_path = f"v3io:///projects/{project_name}/artifacts/{model_name}/"
+
+        def _validate_artifact_tags(expected_tags):
+
+            # get the model from api
+            response = self._run_db.api_call(
+                "GET",
+                f"projects/{project_name}/artifacts?category=model&format=full&best-iteration=true",
+            )
+            assert response.status_code == 200, "failed to get artifacts"
+
+            # we should have 1 artifact per tag
+            artifacts = response.json().get("artifacts", [])
+            assert len(artifacts) == len(expected_tags), "wrong number of artifacts"
+
+            for artifact in artifacts:
+                tag = artifact.get("metadata").get("tag")
+                assert tag in expected_tags, "given tag is not in expected tags"
+
+                # remove the tag from the expected tags
+                expected_tags.remove(tag)
+
+                # make sure artifact doesn't have the model_spec.yaml in spec.extra_data
+                extra_data = artifact.get("spec", {}).get("extra_data")
+                if extra_data:
+                    assert (
+                        "model_spec.yaml" not in extra_data
+                    ), "model spec should not be in extra data"
+
+            # make sure we found all expected tags
+            assert len(expected_tags) == 0, "not all expected tags were found"
+
+        try:
+            pickle_filename = "my_model.pkl"
+            with open(pickle_filename, "wb") as file:
+                pickle.dump({"obj": "my-model"}, file)
+
+            # create a project and log the model
+            project = mlrun.get_or_create_project(project_name, context="./")
```

Comment:
```suggestion
            project = mlrun.get_or_create_project(project_name)
```

context is default to "./" :)